spiceaidocs/docs/local-acceleration/index.md

```diff
@@ -4,3 +4,55 @@ sidebar_label: 'Local Acceleration'
 description: ''
 sidebar_position: 3
 ---
+
+Datasets can be locally accelerated by the Spice runtime, pulling data from any [Data Connector](/data-connectors) and storing it locally in a [Data Accelerator](/data-accelerators) for faster access.
```

Comment:
Is it worth mentioning that some of these connectors can keep the data up to date in realtime? Otherwise a user could read this as, _"we just copy the data from remote, locally"_